src/lib/components/modal-pdp-banner.js

```diff
@@ -2,7 +2,7 @@ import Client from '../client';
 import Charges from '../charges';
 
 const modalPdpBanner = url => {
-  let iframe = `<div className="iframe-container"><iframe scrolling=\"no\" frameBorder=\"0\" id="creditkey-pdp-iframe" src="${url}"></iframe></div>`;
+  let iframe = `<div allowtransparency=\"true\" className="iframe-container"><iframe scrolling=\"no\" frameBorder=\"0\" id="creditkey-pdp-iframe" src="${url}"></iframe></div>`;
```

Comment:
I like how this transparency attr isn't a real W3C thing, but (supposedly) supported by all browsers... https://stackoverflow.com/questions/3740700/there-is-no-attribute-allowtransparency#5182176

but it seems like this attr should be on the `iframe` tag, not the surrounding `div`. I dunno, does this work outside of IE?

Reply:
Good catch, but it's funny that this didn't work even when I moved it to the iframe element. At least not in Chrome. Still needed the CSS changes in ck-marketing to fix this.